src/test/java/org/example/repository/AccountRepositoryImplTest.java

```diff
@@ -0,0 +1,85 @@
+package org.example.repository;
+
+import org.example.connection.ConnectionSingletonTest;
+import org.example.enums.Card;
+import org.example.initDB.InitDBTest;
+import org.example.repository.dao.AccountRepository;
+import org.example.repository.dao.UserRepository;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+import static org.junit.Assert.*;
+
+public class AccountRepositoryImplTest {
+
+    private Connection connection;
+    private AccountRepository accountRepository;
+    private final int testAccountId = 1002;
+    private final int shopAccountId = 1000;
+
+    @Before
+    public void init() {
+        connection = ConnectionSingletonTest.getConnection();
+        InitDBTest.runScript(connection);
+        accountRepository = new AccountRepositoryImpl(connection);
+    }
+
+    @After
+    public void end() throws SQLException {
+        connection.close();
+    }
+
+    @Test
+    public void create() {
```

Comment:
tests name should follow convention given_when_then or WhatIsTestion_WithWhatCondition_WxpectedResult. Without that it takes a lot of time to understand what is the purpose of test 
if test is simple you can skip "WHEN" clause or use only Ok/Error?Failed as expected result